docs/IntegrationGuide.md

```diff
@@ -115,3 +166,88 @@ Example: Let's assume we trigger a subscription-order every 1st of February and
 
 - If the checkout-order was placed until 15 of January then the next subscription-order creation is planned for the 1st of February.
```

Comment:
cuttoffDays -> If the `amount` of days since placing `of` the checkout-order -> If the `number` of days since placing the checkout-order...